deebot_client/events/wash_info.py

```diff
@@ -0,0 +1,23 @@
+"""Wash info event module."""
+from dataclasses import dataclass
+
+from deebot_client.util import DisplayNameIntEnum
+
+from .base import Event
+
+
+class WashMode(DisplayNameIntEnum):
+    """Enum class for all possible wash modes."""
+
+    STANDARD = 0
+    HOT = 1
+
+
+@dataclass(frozen=True)
+class WashInfoEvent(Event):
+    """Wash info event representation."""
+
+    mode: WashMode
+    # None means no data available
+    interval: int
+    hot_wash_amount: int
```

Comment:
```suggestion
    interval: int | None
    hot_wash_amount: int | None
```

If None is a valid value please also add it to typing

Reply:
Sorry, I forgot to remove the comment: it comes from WaterInfoEvent class, which I used as template to code WashInfoEvent. Anyway it is not a bad idea to support also None types (I'm not confident enough that interval and hot_wash_amount are always present)